components/dropdown/dropdown-opener-mixin.js

```diff
@@ -0,0 +1,145 @@
+export const DropdownOpenerMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			isDropdownOpener: {
+				type: Boolean
+			},
+			noAutoOpen: {

```

Comment:
We may need to initialize this to `false` in the constructor.  I think there are certain instances where the linter warn/error if referencing uninitialized booleans.